quantecon/markov/approximation.py

```diff
@@ -212,3 +215,154 @@ def _fill_tauchen(x, P, n, rho, sigma, half_step):
             z = x[j] - rho * x[i]
             P[i, j] = (std_norm_cdf((z + half_step) / sigma) -
                        std_norm_cdf((z - half_step) / sigma))
+
+
+def discrete_var(A,
```

Comment:
Should this function be named `disrete_VAR` given `var` is an acronym for Vector Auto Regression (VAR). `PEP8` has a blanket rule for lower case function names so may not be a great idea. `discrete_var` just reminds me of `discrete variable`.